app/forms/concerns/commencement_date_helpers.rb

```diff
@@ -9,7 +9,13 @@ def commencement_date_valid
     elsif commencement_date < 10.years.ago
       errors.add(:commencement_date, :too_old)
     elsif trainee.course_end_date.present? && commencement_date > trainee.course_end_date
-      errors.add(:commencement_date, I18n.t("activemodel.errors.models.trainee_start_status_form.attributes.commencement_date.not_after_course_end_date", course_end_date: trainee.course_end_date.strftime("%-d %B %Y")))
+      errors.add(
+        :commencement_date,
+        I18n.t(
+          "activemodel.errors.models.trainee_start_status_form.attributes.commencement_date.not_after_course_end_date_html",
+          course_end_date: trainee.course_end_date.strftime("%-d %B %Y"),
+        ),
```

Comment:
```suggestion
        ).html_safe,
```
@darokel this might allow us to fix it without needing a custom presenter
![image](https://user-images.githubusercontent.com/910019/143916660-1e2b787c-c45a-4c9f-919c-9619332b47f3.png)

Reply:
🤦‍♂️